bpf_test.go

```diff
@@ -30,33 +30,33 @@ func TestConstructFilter(t *testing.T) {
 			{Op: 0x0000, Jt: 0x00, Jf: 0x00, K: 0x00000014},
 			{Op: 0x0001, Jt: 0x00, Jf: 0x00, K: 0x00000001},
 			{Op: 0x0030, Jt: 0x00, Jf: 0x00, K: 0xfffff00c},
-			{Op: 0x0015, Jt: 0x0e, Jf: 0x00, K: 0x00000000},
```

Comment:
As the suggested change introduces differences to the expected output from https://git.netfilter.org/libnetfilter_conntrack/tree/utils/conntrack_filter.c I will not apply it for the moment.

Reply:
Ok, I understand.

I like the idea of testing it against the expected output from the original library, but is the produced output always the same?

I have tried to compile and run conntrack_filter.c (master branch) on Ubuntu server and produced output is very different from the go lang version.

What version of the C library do you use as the template for the expected output?

This is how my diff looks like (C version on the left, Go version on the right):
![Screenshot 2023-07-04 at 15 02 40](https://github.com/florianl/go-conntrack/assets/15923956/54cf86c7-95b9-4755-a2cb-70b69021604d)

Reply:
At the moment I'm working on a new testing framework. This should make applying proposed changes easier. But time is a limited resource.

Reply:
In https://github.com/florianl/go-conntrack/pull/50 I have sketched out how bpf testing can be improved. But to run these tests, some changes in golang.org/x/net/bpf are required first.